### PR TITLE
fix(DataTable): Link color should be derived from theme.colors.link

### PR DESCRIPTION
**Pull request by lukelooker**
Update `DataTable` to make plain-old links :)

### Requirements

Please check the following items are addressed in your pull request (or are not applicable)

- [ ] Includes test coverage for all changes
- [ ] Documentation updated
- [ ] i18n impacts
- [ ] a11y impacts

### CODE REVIEW

File: .github/workflows/canary.yml [hunk 1]
```diff
@@ -34,11 +34,10 @@ jobs:
 
       - name: Create canary release
         run: |
-          yarn lerna version prerelease --conventional-commits --preid canary --yes --exact
+          yarn lerna version prerelease --conventional-commits --no-git-tag-version --no-push --preid canary --yes --exact
```
Comment (lukelooker) [inline .github/workflows/canary.yml:37]: Just undoing a change I made that broke things worse.

File: package.json [hunk 2]
```diff
@@ -49,14 +49,16 @@
...
     "storybook": "yarn workspace storybook develop",
     "storybook-docs": "yarn workspace storybook develop-docs",
-    "storybook-workspaces-build": "yarn lerna run storybook-build --scope '@looker/*' --stream --parallel",
 
-    "storyshots-prep": "export storybookBuildMode=fast && yarn pretest && yarn storybook-workspaces-build",
-    "pretest:image-snapshots": "yarn storyshots-prep",
+    "prestoryshots": "yarn prebuild",
+    "storyshots": "export storybookBuildMode=fast && yarn lerna run storybook-build --scope '@looker/*' --stream --parallel",
+
+    "pretest:image-snapshots": "yarn storyshots",
```
Comment (lukelooker) [inline package.json:56]: Make sure storybook is built before taking screenshots

File: package.json [hunk 3]
```diff
@@ -29,8 +29,8 @@
     "build:ts": "yarn lerna exec --stream --scope '@looker/*' --sort 'tsc -b tsconfig.build.json'",
     "bumpversion": "yarn lerna version --conventional-commits --no-git-tag-version --no-push --yes",
     "clean": "run-p -c clean:*",
-    "clean:build": "rm -Rf packages/*/cjs packages/*/esm packages/*/types",
-    "clean:legacy": "rm -Rf packages/*/lib packages/*/dist",
+    "clean:build": "rm -Rf packages/*/lib",
+    "clean:icons": "rm -Rf packages/icons/src/generated",
```
Comment (lukelooker) [inline package.json:33]: Clean wasn't removing generated icons

File: packages/components/src/DataTable/Column/DataTableCell.tsx [hunk 4]
```diff
@@ -136,14 +135,4 @@ DataTableCellLayout.displayName = 'DataTableCellLayout'
...
   ${columnSize}
-  a,
-  ${Link} {
-    color: inherit;
-    :hover,
-    :focus {
-      color: ${({ theme }) => theme.colors.link};
-      outline: none;
-      text-decoration: underline;
-    }
```
Comment (lukelooker) [inline packages/components/src/DataTable/Column/DataTableCell.tsx:147]: This is the meat of the PR here.

File: packages/storybook-config/src/storyshotsConfig.ts [hunk 5]
```diff
@@ -75,7 +75,7 @@ export const imageSnapshots = (pkg: string) => {
         return {
           customSnapshotIdentifier: story,
           customSnapshotsDir: path.join(pkg, 'snapshots', ...kind.split('/')),
-          failureThreshold: 0.05,
+          failureThreshold: 0.005,
```
Comment (lukelooker) [inline packages/storybook-config/src/storyshotsConfig.ts:78]: Make our threshold for failure low (0.5%) instead of huge (5%)

**Review by laraharrow — APPROVED** [review]
LGTM🚢

**Pull request merged**